conans/cli/commands/list.py

```diff
@@ -40,7 +41,7 @@ def list_recipes_cli_formatter(results):
 def _list_revisions_cli_formatter(results, ref_type):
     for remote_results in results:
         _print_common_list_cli_output(remote_results, ref_type)
```

Comment:
This function's name (`_print_common_list_cli_output`) is not very descriptive. How about `_print_remote_header()` or something like that?

Reply:
That function is printing the `Local Cache` line so I did not know what could be the best name... 🤔  Anyway, let me change the var name `remote_results` to `result` because it does not make really sense

Reply:
I'm not sure this function makes a lot of sense. Besides refactoring repeated code, what does it do? Print the header? Handle errors? Show a message when the results are empty?

I mean, if you had to write the docstring of the function, what would it say?

Reply:
It's only supposed to avoid repeating code so I think it makes sense to have it. Perhaps the best name is `_print_common_headers()`, what do you think @tapia ?

Reply:
I've renamed it as `_print_common_headers`